cpu/stm32/periph/eth.c

```diff
@@ -86,6 +98,9 @@ static char rx_buffer[ETH_RX_DESCRIPTOR_COUNT][ETH_RX_BUFFER_SIZE];
 /* Netdev used in RIOT's API to upper layer */
 netdev_t *_netdev;
 
+/* Used for checking the link status */
+static uint8_t _link_state = LINK_STATE_DOWN;
```

Comment:
Compiler complains now about unused variable when module is not not enabled.

Reply:
The variable is now only defined if the module is used.